### PR TITLE
Home page, Single item view, Sell page are made

### PR DESCRIPTION
**Pull request by NihitYadav**

### CODE REVIEW

File: src/Components/Item/Item.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import "./items.css";
+
+function Item(props) {
+  const navigate =useNavigate()
+  function handleClick (){
+    navigate("/product"+'/'+props.id)
```
Comment (aditya-190) [inline src/Components/Item/Item.jsx:8]: Small code improvement -> Use Template literals here.

File: src/Product.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import Navbar from "./Components/Navbar/Navbar";
+import "./Product.css";
+import { useLocation, useParams } from "react-router-dom";
+import data from "./data.json";
+
+function App2() {
+  const location = useLocation()
+  const params = useParams()
+  const product = data.find(item =>item.id==params.id)
+  console.log(product)
```
Comment (aditya-190) [inline src/Product.js:11]: Please remove console.log throughout the code before merging the changes.

File: src/Product.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <>
+    <Navbar />
+      <div className="content2">
+        <img
+          alt="item"
+          src={product.image}
+          className="image"
+        ></img>
+        <div className="details">
+            <div className="price2">{product.price}</div>
```
Comment (aditya-190) [inline src/Product.js:22]: Please give appropriate `classNames` or `varaibleNames` and avoid using names like `prices1`, `prices2`.

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Great Progress. Keep going. Some suggestion have been provided, please make the changes before merging.

**Comment by NihitYadav** [comment]
I am merging this PR for now but I will do the changes that you have suggested in the next PR, Thanks.

**Comment by aditya-190** [comment]
> for now but I will do the changes that you have suggested in t

Sure Nihit.

**Pull request merged**